docs/handleidingen/Publiccode.md

```diff
@@ -87,7 +87,11 @@ nl:
     applicatiefunctie: "referentie component"
 ```
 
-> :note: OpenCatalogi scant GitHub elke nacht, als je een component sneller wilt aanmelden of bijwerken, kan dat via [opencatalogi.nl](https://opencatalogi.nl/documentation/about) gaan en onder "documentatie->over" (hoofd menu). Deze trigger wordt ook afgevuurd door de workflow. Let daarbij op, OpenCatalogi bevraagt hierbij de GitHub Search API, deze geeft alleen goede resultaten terug wanneer de repository is geïndexeerd. Dit kan worden geforceerd door een zoekopdracht uit te voeren op de GitHub website. Gebruik hiervoor de GitHub zoekbalk met bijvoorbeeld `repo:{{Uw Organisatie/Gebruiker}}/{{Uw repository}} publiccode` en wacht tot deze opdracht resultaten geeft. Dit kan zo'n 10 minuten duren en daarbij moet de pagina enige keren ververst worden.
+**Note**
+OpenCatalogi scant GitHub elke nacht, als je een component sneller wilt aanmelden of bijwerken, kan dat door naar [opencatalogi.nl](https://opencatalogi.nl/documentation/about) te gaan en onder "documentatie->over" (hoofd menu). Deze trigger wordt ook afgevuurd door de workflow.
```

Comment:
Ik zou:
" en onder "documentatie->over" (hoofd menu). "
vervangen met iets zoals:
" en daar een component of organisatie in Open Catalogi te registreren door de url van de github repository in te vullen en indienen "